source/sass/buyers-guide/views/product.scss

```diff
@@ -176,6 +176,26 @@
   }
 }
 
+.row.dot-pattern {
```

Comment:
Since the dot pattern is solely for decoration purposes, I would add it in as a pseudo element in CSS like `:before` instead of adding extra DOM nodes just to add background pattern in.

Reply:
done'd